verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -51,7 +49,7 @@ export default function NewCaseForm() {
                             revisionMetadata: {
                                 id: 0,
                                 moderator: "TODO",
-                                date: "",
+                                date: Date.now(),
```

Comment:
This will make the test fail as the test expects the exact data.

The server should fill the revision object, not the client. Which is why I left it empty for now.

Reply:
gotcha, will make sure this happens on the dataserver in a future PR